pkg/exporter/sumologicexporter/config.go

```diff
@@ -114,26 +114,26 @@ func CreateDefaultHTTPClientSettings() confighttp.HTTPClientSettings {
 func (cfg *Config) Validate() error {
 
 	if len(cfg.MetadataAttributes) > 0 {
-		return fmt.Errorf(`metadata_attributes is not supported anymore.
-Please consult the changelog at https://github.com/SumoLogic/sumologic-otel-collector/releases/tag/v0.49.0-sumo-0`,
+		return fmt.Errorf(`the property metadata_attributes was removed in v0.49.0-sumo-0.
+See upgrade guide at https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/upgrading.md#sumo-logic-exporter-metadata-handling`,
 		)
 	}
```

Comment:
Can't we just remove it? It was `23` version back

Reply:
Yeah we probably should get rid of these warnings at one time or another :smile: